app/views/support_interface/application_forms/courses/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, 'Pick a course' %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_search_course_new_path) %>
```

Comment:
Could we feed in the course code to the back link as a query string, so that clicking back retains the search term?

```suggestion
<% content_for :before_content, govuk_back_link_to(support_interface_application_form_search_course_new_path(course_code: @course_code) %>
```